.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,27 +1,38 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Suggest an idea for a new feature
 title: ""
-labels: "enhancement"
```

Comment:
`enhancement` was a default GitHub label (that we've already used in the past as well), so I think it might be best to leave it as-is, in case GitHub does clever things with it. (For example, it also has a `good first issue` label, and it has pages where it directs people who want to contribute to open source to projects that use that label.)